src/main/java/controller/DbConnection.java

```diff
@@ -0,0 +1,46 @@
+package controller;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class DbConnection {
```

Comment:
You don't need this class at all. You should create Connection when you need it, so remove it